src/test/java/org/magicdgs/readtools/utils/trimming/TrimAndFilterPipelineUnitTest.java

```diff
@@ -583,7 +585,7 @@ public void testGetPipelineCompatibleWithGATKReadFilterPluginDescriptor(
 
         // get the trimming pipeline arguments
         final TrimAndFilterPipeline pipeline = TrimAndFilterPipeline.fromPluginDescriptors(
-                new TrimmerPluginDescriptor(defaultTrimmers),
+                new TrimmerPluginDescriptor(new TrimReadsTrimmerPluginArgumentCollection(), defaultTrimmers),
```

Comment:
Here it will be much better to keep only the test for the `GATKReadFilterPluginDescriptor` instead.

Reply:
Will do with #351